feature-group-definitions/line-clamp.yml

```diff
@@ -1,5 +1,16 @@
 spec: https://drafts.csswg.org/css-overflow-3/#line-clamp
 caniuse: css-line-clamp
+status:
+  baseline: high
```

Comment:
Since this requires a prefix, but `line-clamp` will probably also be shipped at some point, I think this one requires a note. Split into a separate PR?

Reply:
Moved to https://github.com/web-platform-dx/web-features/pull/503.